pallets/delegation/src/lib.rs

```diff
@@ -323,25 +326,30 @@ impl<T: Trait> Module<T> {
 	pub fn is_delegating(
 		account: &T::AccountId,
 		delegation: &T::DelegationNodeId,
+		max_depth: u64,
 	) -> result::Result<bool, &'static str> {
+		if max_depth == 0 {
+			return Err("Reached max delegation search depth");
+		}
+
 		// check if delegation exists
 		let d = <error::Module<T>>::ok_or_deposit_err(
```

Comment:
Still wondering whether we should give a more distinct name?

Reply:
`ok_or_deposit_err`? I think this will be touched when we refactor errors?

Reply:
ahhhhh you meant the `d`... fixed in 951c5da